python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -327,6 +334,10 @@ def get_partition_keys(
         )
         check.opt_int_param(start_idx, "start_idx")
         check.opt_int_param(end_idx, "end_idx")
+
+        if not self._mutable_partitions_loader:
+            check.failed("mutable_partitions_loader must be provided to get partition keys")
```

Comment:
Is this string right?

Reply:
Yes, the intent is so that when loading asset graphs / asset jobs any asset nodes all have a shared instance of the mutable partitions loader. I think I've added a mutable partitions loader to every asset node callsite, so nobody should run into this error